test/unit/test-layout.cxx

```diff
@@ -53,7 +54,7 @@ TEST(LayoutTest, 2D_IJ)
    * (-1, -1), (0, -1), (1, -1)
    * (-1, -2), (0, -2), (1, -2)
    */
-  const my_layout layout = my_layout::from_arrays({-1,-2}, {1,0});
+  const auto layout = RAJA::make_offset_layout<2>({-1, -2}, {1, 0});
```

Comment:
I think it's perfectly reasonable to specify the dimensionality. It also helps with readability so the user doesn't need to start counting commas/braces